Signal/src/util/DateUtil.m

```diff
@@ -357,46 +357,50 @@ + (NSString *)formatMessageTimestamp:(uint64_t)timestamp
 
     NSCalendar *calendar = [NSCalendar currentCalendar];
 
+    NSDateComponents *relativeDiffComponents =
```

Comment:
Calculate the "relative time difference" minutes and hours as a single calculation, not as two separate calculations.